src/js/actions/layereffects.js

```diff
@@ -325,6 +333,89 @@ define(function (require, exports) {
     setShadowSpread.reads = [locks.PS_DOC, locks.JS_DOC];
     setShadowSpread.writes = [locks.PS_DOC, locks.JS_DOC];
 
+    /**
+     * Copies the layer effects of the source layer to all the layers
```

Comment:
I don't think "copy" is the right word here. Maybe "transfer" would be clearer? Just kicking ideas around.